### DISCUSSION TITLE
Nested plugins.sbt file not getting used

### DISCUSSION THREAD

**Discussion by yadavan88** [sbt/sbt#7325] [Q&A] [answered]
Hi there,
I have a project structure like this:

```
root
project
    | - plugins.sbt
grouping
   | - play-module1
             | - build.sbt
             | - project
                      | -  plugins.sbt (added play plugin)
```

In this case, If the play plugin is not added in the root project's plugins.sbt, then I am getting error while enabling play plugin in the play-module's build.sbt. However, it works if the addPlugin is moved to the root plugin.sbt.
Is this disallowed? I thought plugins will also be handled in the same way as nested build.sbt (with merging and scoping). Is it not the case?

Thanks in advance

**Comment by eed3si9n** [comment, marked as answer]
Not really. Only `project/` can load plugins.
